tools/capture-test.py

```diff
@@ -111,8 +114,18 @@ def pcm_init_stream(pcm, rate, chans, fmt, access):
     alsa.snd_pcm_hw_params_any(pcm, hwp)
     alsa.snd_pcm_hw_params_set_format(pcm, hwp, fmt)
     alsa.snd_pcm_hw_params_set_channels(pcm, hwp, chans)
-    alsa.snd_pcm_hw_params_set_rate(pcm, hwp, rate, alsa.PCM_STREAM_PLAYBACK)
+    alsa.snd_pcm_hw_params_set_rate(pcm, hwp, rate, 0)
```

Comment:
I don't understand how this parameter change is related to this commit but it's your script so I'll just trust you on that.

Reply:
It's not, it's a misuse fixup I'd forgotten about.  That argument ("dir") isn't specifying the direction of the stream, it's specifying the "direction" (in the set -1, 0, 1) in which the set command will err if it doesn't find an exact match.  It was a silent noop because (1) there is an exactly match in practice at 48kHz and (2) PCM_STREAM_PLAYBACK has the value zero anyway.  Will split.